src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectDefaults.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+
```

Comment:
nit: blank line added

Reply:
Whoops, that must have happend while adding and removing stuff in differerent commits. Will be gone when I squash the changes.